### PR TITLE
Added JOIN command.

### PR DESCRIPTION
**Pull request by ThatIsAUsername**

### CODE REVIEW

File: src/Engine/GameAction.java [hunk 1]
```diff
@@ -610,6 +611,93 @@ public String toString()
...
+      isValid &= (null != movePath) && (movePath.getPathLength() > 0);
+      isValid &= (null != gameMap);
+      if( isValid )
+      {
+        pathEnd = new XYCoord(movePath.getEnd().x, movePath.getEnd().y);
+        isValid &= gameMap.isLocationValid(pathEnd);
+
+        if( isValid )
+        {
+          // Find the transport unit.
```
Comment (ThislsAUsername) [inline src/Engine/GameAction.java]: "transport"

**Comment by ThislsAUsername** [comment]
It does print a warning about being unable to move when you join, but it's probably not a big deal.
...especially as I'm planning to muck around with everything soon.

**Comment by ThatIsAUsername** [comment]
> It does print a warning about being unable to move when you join, but it's probably not a big deal.
> ...especially as I'm planning to muck around with everything soon.

Yeah, it does the same thing for load actions. Which I inadvertently broke, but it's fixed now.

**Comment by ThislsAUsername** [comment]
> but it's fixed now.
Still printing the warning on my end, and I made sure to pull.

**Comment by ThatIsAUsername** [comment]
Sorry, wasn't clear. Load commands have always printed that warning. No intention to fix that at this time.

I did however, accidentally make it impossible to put a unit on a transport in the initial PR. That's what I fixed.

**Pull request merged**